### PR TITLE
CATROID-758 Copy look brick

### PR DESCRIPTION
**Pull request by blindguardian50**
The brick copies the current look and inserts it in the looklist.

https://jira.catrob.at/browse/CATROID-758

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by dandrawes — APPROVED** [review]
Well done!

File: catroid/src/main/java/org/catrobat/catroid/common/BrickValues.java [hunk 1]
```diff
@@ -67,6 +67,7 @@
 	public static final int GO_TO_RANDOM_POSITION = 81;
 	public static final int GO_TO_OTHER_SPRITE_POSITION = 82;
 	public static final int SET_LOOK_BY_INDEX = 1;
+	public static final String COPY_LOOK = "name of copied look";
```
Comment (wslany) [inline catroid/src/main/java/org/catrobat/catroid/common/BrickValues.java:70]: This should be a translatable resource in strings.xml

**Review by wslany — CHANGES_REQUESTED** [review]
Thanks!!

How complex is it that looks imported with a "Get image from 'https://catrob.at/penguin' and use as current look" can also be copied (and then become a real, normal look) with the new brick? This would be very useful, and also would be very natural behavior. If it is very complex, we can also make a bug ticket and handle it there later. LMK.

File: catroid/src/main/java/org/catrobat/catroid/content/actions/LookRequestAction.kt [hunk 2]
```diff
@@ -48,14 +48,27 @@ open class LookRequestAction : WebAction() {
     }
 
     override fun handleResponse() {
-        val look = getLookFromResponse()
-        look?.apply {
-            sprite!!.look?.lookData = this
+        val lookData = getLookFromResponse()
+        lookData?.apply {
+            insertInLookList(this)
```
Comment (dandrawes) [inline catroid/src/main/java/org/catrobat/catroid/content/actions/LookRequestAction.kt:53]: If a "Get image from" brick is not used in combination with the "Copy look" brick a new look is created, which should not be the case.

**Review by dandrawes — CHANGES_REQUESTED** [review]

**Review by dandrawes — APPROVED** [review]
Good job, looks good now!

**Review by wslany — CHANGES_REQUESTED** [review]
Thanks!! 

Please increase the Catrobat language version in https://github.com/Catrobat/Catroid/blob/develop/catroid/src/main/java/org/catrobat/catroid/common/Constants.java#L42 (please do not increase to 1.0 --- this is reserved to when the user-defined brick functionality is merged, which hopefully will be very soon).

I'm keeping this ticket in PO review since this a small change and no second code-review is necessary. Please reply to this comment to let me know when I can merge it. Thanks!

**Comment by blindguardian50** [comment]
Thanks for the review, I have updated the language version, you can merge it now!

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**